opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -375,12 +375,21 @@
                     "contentVersion": "1.0.0.0"
                 },
                 "parameters":{  
+                    "_artifactsLocation": {
+                        "value": "[parameters('_artifactsLocation')]"
+                    },
+                    "_artifactsLocationSasToken": {
+                        "value": "[parameters('_artifactsLocationSasToken')]"
+                    },
                     "applicationGatewaySubnetId": {
                         "value": "[reference('vnet').outputs.appgwSubnetId.value]"
                     },
                     "dnsNamePrefix":{
                         "value": "[variables('dnsNamePrefix')]"
                     },
+                    "appGwSslEmail": {
+                        "value": "[parameters('mongoCertificateEmail')]"

```

Comment:
why is this using the MONGO cert email? Not saying that's the wrong thing to use, but does it suggest that "mongoCertificateEmail" is the wrong name for this parameter?

Reply:
Agreed. Still think one email address for both the mongodb & appgw ssl certificates is sufficient. But renamed the param so it makes more sense @https://github.com/opencb/opencga/pull/1202/commits/40a78ca90e6f746f8bf632fb7dbe302d7f626c70

Reply:
agreed!

Reply:
Unified email addresses in https://github.com/opencb/opencga/pull/1202/commits/40a78ca90e6f746f8bf632fb7dbe302d7f626c70